### PR TITLE
fixed slack

### PR DESCRIPTION
**Pull request by pythexcel**

### CODE REVIEW

File: app/api/monthly.py [hunk 1]
```diff
@@ -6,11 +6,11 @@
 )
```
Comment (xmagestore) [inline app/api/monthly.py]: one important this is missing i.e removal of slack module from requuirement.txt .

File: app/api/report.py [hunk 2]
```diff
@@ -130,12 +84,27 @@ def add_checkin():
...
-                slack_msg(channel=slackChannels,
-                          msg="<@" + slack + ">!" + "\n" + "Report: " + "\n" + slackReport + "" + "\n"
-                              + "Highlight: " + highlight)
+                data = "Report: " + "\n" +slackReport + "" + "\n" + "Highlight: " + highlight
+                check_in_payload = {
+                    "user": {
+                        "email": current_user['work_email']},
+                        "data": data,
+                        "message_type" : "simple_message",
+                        "message_key": "check-in"
```
Comment (xmagestore) [inline app/api/report.py]: why is message_key and type comming inside user?? this doesn't make sense. it should be outside. for user we need email, name, emp_id and phone. all are missing

File: app/config.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 URL = 'http://176.9.137.77/hr_tms/ReactReduxHR/backend/attendance/API_HR/api.php'
 URL_details = 'http://176.9.137.77/hr_tms/ReactReduxHR/backend/attendance/sal_info/api.php'
+nt_URl = 'https://still-plains-80981.herokuapp.com/notify/dispatch'
 #secret_key = '3dd7fe8a6ea2ea9afb9a7366980253b7'
 default={
             "monthly_remainder":"Slack_id: Please create your monthly report of :Month till 10th of this month. Failing to do so will automatically set your monthly review to 0.",
```
Comment (xmagestore) [inline app/config.py]: all these defaults messages should be removed

File: app/config.py [hunk 4]
```diff
@@ -1,5 +1,6 @@
 URL = 'http://176.9.137.77/hr_tms/ReactReduxHR/backend/attendance/API_HR/api.php'
 URL_details = 'http://176.9.137.77/hr_tms/ReactReduxHR/backend/attendance/sal_info/api.php'
+nt_URl = 'https://still-plains-80981.herokuapp.com/notify/dispatch'
```
Comment (xmagestore) [inline app/config.py]: make this notification_system_url

File: app/scheduler.py [hunk 5]
```diff
@@ -123,14 +124,17 @@ def monthly_remainder():
...
                     today_date = int(today.strftime("%d"))
                     #check if user allow date(joiningdate + 10) is greater then to today date then send normal slack msg else create a report with default ratings
-                    if today_date<11:
+                    if today_date<27:
                         if role != 'Admin':
-                            monthly_mesg=mesg.replace("Slack_id:", "<@" + slack_id + ">!")
-                            msg=monthly_mesg.replace(":Month",""+str(month)+"")
-                            slack_message(msg=msg)
+                            monthly_reminder_payload = {"user":{"email":email},"data":month,"message_key":"monthly_reminder","message_type":"simple_message"}
+                            slack_message = requests.post(url=nt_URl,json=monthly_reminder_payload)
```
Comment (xmagestore) [inline app/scheduler.py]: change this to notification_message everywhere

File: app/api/report.py (hunk 2 above)
Comment (pythexcel) [reply app/api/report.py]: it is not sir the vs code formatter made the structure after current_user['work_email'] } this is there to close user dict ok will add name , emp_id  and phone btw phone is not there in out data structure

File: app/config.py (hunk 3 above)
Comment (pythexcel) [reply app/config.py]: ok will removed them

File: app/config.py (hunk 4 above)
Comment (pythexcel) [reply app/config.py]: ok will make notification_system_url

File: app/scheduler.py (hunk 5 above)
Comment (pythexcel) [reply app/scheduler.py]: ok sir will change this to notification_message

File: app/api/monthly.py (hunk 1 above)
Comment (pythexcel) [reply app/api/monthly.py]: ok sir removing will be done

**Pull request merged**